packages/ckeditor5-table/src/tablecaption/tablecaptionediting.js

```diff
@@ -10,7 +10,10 @@
 import { Plugin } from 'ckeditor5/src/core';
 import { enablePlaceholder } from 'ckeditor5/src/engine';
 import { toWidgetEditable } from 'ckeditor5/src/widget';
+
 import injectTableCaptionPostFixer from '../converters/table-caption-post-fixer';
+import ToggleTableCaptionCommand from './toggletablecaptioncommand';
+import { isTable, matchTableCaptionViewElement } from './utils';
```

Comment:
Let's align this with the image caption https://github.com/ckeditor/ckeditor5/blob/i/2052-inline-images/packages/ckeditor5-image/src/imagecaption/imagecaptionediting.js#L121-L126